web_programming/get_top_billionaires.py

```diff
@@ -20,18 +20,29 @@
 )
 
 
-def calculate_age(unix_date: int) -> str:
+def calculate_age(unix_date: float) -> str:
     """Calculates age from given unix time format.
 
     Returns:
         Age as string
 
     >>> calculate_age(-657244800000)
-    '73'
+    '74'
     >>> calculate_age(46915200000)
-    '51'
+    '52'
```

Comment:
Why change the doctest outputs? Were they originally off by 1?

Reply:
Yes, the age of the timestamps increased 😆 Interesting doctest because each year the ages will have to be incremented

Reply:
Ok that's really not ideal... Can we modify these tests so that the ages automatically update? Perhaps by converting dummy datetime objects to UNIX timestamps and passing those into the function